build.gradle

```diff
@@ -14,15 +14,10 @@
  * limitations under the License.
  */
 
-buildscript {
-  repositories {
-    jcenter()
-    maven { url "https://dl.bintray.com/spinnaker/gradle" }
-    maven { url "https://plugins.gradle.org/m2/" }
-  }
-  dependencies {
-    classpath 'com.netflix.spinnaker.gradle:spinnaker-gradle-project:3.15.0'
-  }
+plugins {
+  id 'base'
+  id "io.spinnaker.project" version "$spinnakerGradleVersion" apply false
```

Comment:
with 8.0.5 this can be id "io.spinnaker.bintray-publish"

I still think apply false makes sense in this repo because the deb publishing is in the subprojects..